examples/caller/make_multiple_calls.py

```diff
@@ -0,0 +1,125 @@
+import threading
+from threading import Thread
+from typing import List
+
+from ondewo.nlu import context_pb2
+from ondewo.vtsi.client import ConfigManager
+from ondewo.vtsi.dataclasses.server_configurations_dataclasses import (
+    AsteriskConfiguration,
+    AudioConfiguration,
+    CaiConfiguration,
+    VtsiConfiguration,
+)
+from ondewo.vtsi.voip_pb2 import StartCallInstanceResponse
+
+##### Globals #####
+
+#     SIP
+SIP_SIM_VERSION: str = "1.5.4"
```

Comment:
can you just use the 'get_minimal_client' function, and make another script called 'configure_vtsi_server.py' where you show off all this config stuff?

I would like to end up with a few scripts, each demonstrating a simple idea-- this one should just have the multiple calls material in it